docs/UsersGuide/source/SRWAppOverview.rst

```diff
@@ -54,60 +53,59 @@ The cloned repository contains the configuration files and sub-directories shown
    +--------------------------------+--------------------------------------------------------+
-   | docs                           | Release notes, documentation, User's Guide             |
+   | docs                           | Contains Release notes, documentation, and Users' Guide|
    +--------------------------------+--------------------------------------------------------+
-   | manage_externals               | Method for checking out external components            |
+   | manage_externals               | Utility for checking out external repositories         |
    +--------------------------------+--------------------------------------------------------+
-   | src                            | Contains CMakeLIsts.txt; the external components       |
+   | src                            | Contains CMakeLists.txt; external repositories         |
    |                                | will be cloned in this directory.                      |
    +--------------------------------+--------------------------------------------------------+
 
 .. _CheckoutExternals:
 
 External Components
 ===================
-Check out the sub-modules such as regional_workflow, ufs_weather_model, ufs_utils, and emc_post for SRW App.
+Check out the external repositories, including regional_workflow, ufs-weather-model, ufs_utils, and emc_post for the SRW App.
 
 .. code-block:: console
 
    ./manage_externals/checkout_externals
 
 This step will use the configuration ``Externals.cfg`` file in the ``ufs-srweather-app`` directory to
-clone the specific hashes (version of codes) of the external components as listed in 
+clone the specific hashes (version of codes) of the external repositories as listed in 
 :numref:`Section %s <HierarchicalRepoStr>`. 
 
 .. _BuildExecutables:
 
 Building the Executables for the Application
 ============================================
-Before building the executables, the build environment must be set up for your individual platform.
-Instructions for loading the proper modules and/or setting the correct environment variables for
-can be found in the ``docs/`` directory in files named ``README_<platform>_<compiler>.txt.`` For the
-most part the commands in those files can be directly copy-pasted, but you may need to modify
-certain variables such as the path to NCEP libraries for your individual platform.  The commands
-are in the following files:
+Before building the executables, the build environment must be set up for your specific platform.
+Instructions for loading the proper modules and/or setting the correct environment variables 
+can be found in the ``docs/`` directory in files named ``build_<platform>_<compiler>.env.`` For the
+most part, the commands in those files can be directly copied and pasted, but you may need to modify
+certain variables such as the path to NCEP libraries for your specific platform.  Here is a directory
+listing example of these kinds of files: 
 
 .. code-block:: console
 
    $ ls -l docs/
```

Comment:
This is in `env`, not in `docs`, correct?

Reply:
Correct!  Thanks for catching that.

Reply:
Line 103 in docs/UsersGuide/source/SRWAppOverview.rst is still `docs` (sorry if you are still working on it)

Reply:
Thanks, fixed that one too!